response_operations_ui/templates/reporting-unit.html

```diff
@@ -16,133 +16,146 @@ <h1 id="RU_NAME" class="jupiter">{{ ru.name }}</h1>
 
   <h2 class="saturn">Surveys</h2>
 
-  {% for survey in surveys %}
-  <div class="ru-survey" name="associated-surveys">
-    <h3 name="survey-titles" class="saturn">{{ survey.surveyRef }} {{ survey.shortName }}</h3>
+  <section class="grid__col col-12@l ru-surveys">
+    {% for survey in surveys %}
+    <div class="ru-survey data-panel" id="survey-{{survey.shortName}}" name="associated-surveys">
+      <h3 name="survey-titles" class="saturn data-panel-header">{{ survey.surveyRef }} {{ survey.shortName }}</h3>
```

Comment:
Missing an onclick=openPanel()? currently the panel toggle feature does not work. Feels like the the panel toggle feature should of been part of another PR.

Reply:
Potentially, I've just been trying to keep the UI up to date with the prototype while I've been adding functionality. I'll have a look into the javascript today and see if I can clean it up easily, I think there's a lot in there atm that we don't need